integreat_cms/cms/forms/push_notifications/push_notification_form.py

```diff
@@ -22,6 +42,62 @@ def __init__(self, **kwargs):
             self.fields["channel"].disabled = True
             self.fields["mode"].disabled = True
 
+        # Set the day and time fields
+        if self.instance.scheduled_send_date:
+            local_time = self.instance.scheduled_send_date_local
+            self.fields["scheduled_send_date_day"].initial = local_time.date()
+            self.fields["scheduled_send_date_time"].initial = local_time.time()
+            self.fields["schedule_send"].initial = True
+
+    def clean(self):
+        """
+        Validate form fields which depend on each other, see :meth:`django.forms.Form.clean`
+
+        :return: The cleaned form data
+        :rtype: dict
+        """
+        cleaned_data = super().clean()
+
+        if cleaned_data.get("schedule_send") and not cleaned_data.get(
+            "scheduled_send_date_day"
+        ):
+            self.add_error(
+                "scheduled_send_date_day",
+                forms.ValidationError(
+                    _("Cannot be unspecified"),
+                ),
+            )
+
+        # Combine the scheduled send day and time into one timezone aware field
+        if not self.errors and cleaned_data.get("schedule_send"):
+            tzinfo = zoneinfo.ZoneInfo(self.instance.timezone)
+            time = cleaned_data["scheduled_send_date_time"] or datetime.time()
+
+            cleaned_data["scheduled_send_date"] = datetime.datetime.combine(
+                cleaned_data["scheduled_send_date_day"], time
+            ).replace(tzinfo=tzinfo)
+
+            if cleaned_data["scheduled_send_date"] < timezone.now():
```

Comment:
To my understanding, no field should be highlighted in that case currently. Could you tell me how to reproduce that behavior?

Reply:
I think I just opened a PN, set the scheduled time to today but earlier than now (e.g. 12:00 currently) and saved. I believe the date field got highlighted in red.

Reply:
The only place that highlights the date should be this: https://github.com/digitalfabrik/integreat-cms/pull/2351/files#diff-b9112fcbea577d16c5ea50b79f1044e02a5f77ae04913e4f45c6fb6ec4999c69R78. 
So currently no field would be highlighted if the date is in the past. Do you think this should be added anyways?

Reply:
True, I cannot reproduce my observation with the current state. If this was my own project I would probably implement it even if it didn't have any significance for users using the interface as intended and not deleting the `min=` to try and get around the restriction, but I think it can be considered dead weight